1/Dockerfile

```diff
@@ -48,9 +48,11 @@ RUN /usr/local/bin/plugins.sh /opt/openshift/base-plugins.txt && \
     touch /opt/openshift/plugins/subversion.jpi.pinned && \
     touch /opt/openshift/plugins/ssh-credentials.jpi.pinned && \
     touch /opt/openshift/plugins/script-security.jpi.pinned && \
+    chmod 775 /var/log/jenkins && \
```

Comment:
I'll have to figure out how `/etc/logrotate.d/jenkins` is getting created and based on that see if we can modify.  Assuming we can say manipulate it in the dockerfiles, then yes I think we can make this change.